third-party/cmake/LibvtermCMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(libvterm LANGUAGES C)
```

Comment:
Not sure upstream is interested in a cmake recipe.

I've kept a separate file just to avoid making the recipe more confusing. But I can merge everything into a single file.